index.js

```diff
@@ -4,26 +4,22 @@ const util = require('util');
 const path = require('path');
 const { Pair } = require('yaml/types');
 
-const { possibleLocations } = require('./src/helpers');
+const {
+  alacrittyFileExists,
+  alacrittyConfigPath,
+  NoAlacrittyFileFoundError,
+} = require('./src/helpers');
 
 const readFile = util.promisify(fs.readFile);
 const writeFile = util.promisify(fs.writeFile);
 
-const noConfigErr = new Error(
-  'No configuration file for alacritty found. Expected one of the following files to exist:\n' +
-    possibleLocations().join('\n')
-);
-
+// pick the correct config file or handle errors, if it doesn't exist
 function getAlacrittyConfig() {
-  // pick the correct config file or handle errors, if it doesn't exist
-  function findExistingFile(files) {
-    for (let configPath of files)
-      if (fs.existsSync(configPath)) return configPath;
-
-    throw noConfigErr;
+  if (!alacrittyFileExists()) {
+    throw NoAlacrittyFileFoundError;
```

Comment:
since we know when the user hasn't an Alacritty config file, I was wondering if we can show a prompt to ask them `Alacritty file doesn't exist, would I create it?` or something like that, and then create it.

Reply:
It was like that earlier, I don't remember why I removed it. Yes we can show a prompt to create the config file

Reply:
I'll do it in another PR, I don't want to break things.

Reply:
This is ready to be merged if you agree 👍